src/olympia/lib/tests/test_git.py

```diff
@@ -38,6 +39,24 @@ def test_temporary_worktree():
     assert worktree.name not in output
 
 
+def test_enforce_pygit_global_search_path_to_home():
+    # https://github.com/mozilla/addons-server/issues/10320
+    pygit2.settings.search_path[pygit2.GIT_CONFIG_LEVEL_GLOBAL] = '/root'
```

Comment:
Shouldn't you revert this to the original value ? And also make sure that it is indeed reverted, regardless of whether the test succeeds or not ?

Reply:
indeed, I'll fix that.

Reply:
Fixed in 9b2d5a5